landingpage.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+<body>
+<div class="header">
+<div class="original">
+   <div class="logo"> <img src="product-landing-page-logo.png" alt=""></div>
+</div>
+<div class="headerlink">
+    <a href="">Features</a>
+    <a href="">How it works</a>
+    <a href="">Pricing</a>
+</div>
+
+</div>
+<div class="hero">
+    <div class="handcrafted">
+       <h2>Handcrafted,home-made masterpieces</h2>
+    </div>
+    <div class="placeholder">
+        Enter your email address
+    </div>
+    <button class="get" > Get started</button>
+</div>
+<div class="container">
+    <div class="premium">
+      <h1>  Premium Materials</h1> 
+      <div class="smallpremium">Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</div>
+
+    </div>
+    <div class="premium">
+        <h1>Fast Shipping</h1>
+        <div class="smallshipping">We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</div>
+
+    </div>
+    <div class="premium">
+        <h1>Quality Assurance</h1>
+        <div class="smallquality">
+            Quality Assurance
+            For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</div>
+    </div>
+
+</div>
```

Comment:
Please use more Semantic Tags instead of div tags
Examples: section, header article ... Tags

Reply:
- [ ]  Please use a couple of emojis and/or gifs in your review, they make the code review more human-friendly 👽🤖.
- [ ] Also explain clearly why the student needs to use the semantic tags 🙃 😉